azure-kusto-data/azure/kusto/data/_telemetry.py

```diff
@@ -0,0 +1,158 @@
+from typing import Callable, Optional
+
+from azure.core.settings import settings
+from azure.core.tracing.decorator import distributed_trace
+from azure.core.tracing.decorator_async import distributed_trace_async
+from azure.core.tracing import SpanKind
+
+from .client_request_properties import ClientRequestProperties
+
+
+class KustoTracingAttributes:
+    """
+    Additional ADX attributes for telemetry spans
+    """
+
+    _KUSTO_CLUSTER = "Kusto Cluster"
```

Comment:
Why are some Upper case with spaces and some lowercases with dots?

Reply:
fixed